service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ResourceTypeCodes.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+public final class ResourceTypeCodes {
+
+    private ResourceTypeCodes() {
+        // hiding the constructor
+    }
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class ResourceTypeCodes {

```
lombok annotation could be used instead